robot/game_config/markers.py

```diff
@@ -97,14 +108,20 @@ def by_id(id: int, team: typing.Union[TEAM, None] = None) -> BASE_MARKER: # team
         there). In practice these markers start at 100.
         """
 
-        ARENA_WALL_LOWER_ID = 40
-        if id >= ARENA_WALL_LOWER_ID:
+        if id > 99:
             return ARENA_MARKER(id)
-        
-        wrappingId = id % 20 # Make sure that the ID range wraps after 20 values.
-        if wrappingId<4: # If it is a Jacket Potato (has a team)
-            owning_team = TEAM[f"T{wrappingId}"] # Set to the corresponding TEAM enum.
-        else: # If it is a Hot Potato (Has no team)
-            owning_team = TEAM["ARENA"]
 
-        return POTATO_MARKER(id, owning_team)
+        if id < 24:
+            owning_team = TEAM["ARENA"]
+        elif id == 30 or id == 31 or id == 53:
+            owning_team = TEAM["T3"]
+        elif id == 28 or id == 29 or id == 52:
+            owning_team = TEAM["T2"]
+        elif id == 26 or id == 27 or id == 51:
+            owning_team = TEAM["T1"]
+        elif id == 24 or id == 25 or id == 50:
+            owning_team = TEAM["T0"]
+        else:
+            raise Exception("AHHHHHH!")
```

Comment:
It would help teams to make this clearer about what exactly has gone wrong.

Reply:
That was left over during testing for undefined marker IDs. This has now been replaced with a new owning team "NONE". This will probably never occur during a game unless markers are used that are not within the arena range or other defined markers 0-25 and 50-53.